tutorials/hdfs-sync/src/site/conf/cluster-memory-conf.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<configuration>
```

Comment:
Currently, launch UI has no way to separate out mandatory properties and advanced properties.
Thus, as a workaround we have the following strategy:
1. All the properties for which end user must specify the values (e.g. input path, output path etc.) are part of properties.xml.
2. All other properties are (for which we can set reasonable defaults) are part of apconf.
3. apconf have two profiles: 
    a. cluster which is suitable for medium to big cluster deployment.
    b. sandbox which is suitable for RTS sanbox / single node deployment.